Sources/StreamChat/Controllers/PollController/PollVoteListController.swift

```diff
@@ -126,6 +128,10 @@ public class PollVoteListController: DataController, DelegateCallable, DataStore
 
         pollsRepository.queryPollVotes(query: query) { [weak self] result in
             guard let self else { return }
+            if let value = result.value {
+                self.nextCursor = value.next
+                self.hasLoadedAllVotes = value.votes.count < query.pagination.pageSize
```

Comment:
I actually had a bug on Threads v2 where because the current user was removed from those threads, the page did not return the `limit` that it should have. So it is actually safer we check the `next` value to determine if there are more data or not. (This is a bug on the backend, but better to be safer)
```suggestion
                self.hasLoadedAllVotes = value.next == nil
```

Reply:
Done